### PR TITLE
Add page for creating an event plan to web2

### PR DESCRIPTION
**Pull request by jesc0bar**

### CODE REVIEW

**Comment by mtahery20** [comment]
Looks good! Functionality is working too (invited myself to an event)

**Review by mtahery20 — DISMISSED** [review]

**Comment by navarrojandg** [comment]
Looks good to me -- code wise. Just wanted to note that my approval doesn't carry much weight for web2 related PRs vs other members who are working on web2.

@ivancuadra869 Just wanted to note here that Jorge has started to bring in the `web/src/modules` folder into `web2`

**Review by navarrojandg — DISMISSED** [review]

File: apps/web2/src/pages/PlanAnEventPage/PlanAnEventPage.tsx [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+
+  return (
+    <Sidebar>
+      <main className="max-w-lg mx-auto pt-20 pb-12 px-4 lg:pb-16">
+        <form onSubmit={onSubmitHandler}>
+          <input type="hidden" name="hostId" value={user?.uid} />
+          <div className="space-y-6">
+            <div>
+              <h1 className="text-lg leading-6 font-medium text-gray-900">
+                Event Settings
```
Comment (ivancuadra869) [inline apps/web2/src/pages/PlanAnEventPage/PlanAnEventPage.tsx]: Not sure if you meant to change this, but it seems like it should be something else.

**Review by ivancuadra869 — CHANGES_REQUESTED** [review]
Looks great and the functionality also works.

**Pull request merged**